Unable to find image '665840871993.dkr.ecr.us-east-1.amazonaws.com/dev_tools:prod' locally
prod: Pulling from dev_tools
7c3b88808835: Already exists
ecc05375b50a: Already exists
7804defa212a: Already exists
c524d90d3a1c: Already exists
4530ca4a6aa0: Pulling fs layer
7b0c919d07c0: Pulling fs layer
be363baee980: Pulling fs layer
c50686409e37: Pulling fs layer
90f081dbcb3b: Pulling fs layer
b9d9a8a3036e: Pulling fs layer
466b9fe9cca0: Pulling fs layer
b56689c0ae24: Pulling fs layer
69c36f50f93d: Pulling fs layer
7863da9f2274: Pulling fs layer
e1215d28dca8: Pulling fs layer
1f4b25b96394: Pulling fs layer
02013e989733: Pulling fs layer
66fe70d75676: Pulling fs layer
f4d1a20d6c1a: Pulling fs layer
a49e148c97b0: Pulling fs layer
d65a4f30a4fa: Pulling fs layer
b17d3e90cf7f: Pulling fs layer
3ca8e3307e40: Pulling fs layer
90f081dbcb3b: Waiting
69c36f50f93d: Waiting
b9d9a8a3036e: Waiting
7863da9f2274: Waiting
d65a4f30a4fa: Waiting
66fe70d75676: Waiting
b17d3e90cf7f: Waiting
f4d1a20d6c1a: Waiting
3ca8e3307e40: Waiting
a49e148c97b0: Waiting
e1215d28dca8: Waiting
466b9fe9cca0: Waiting
1f4b25b96394: Waiting
b56689c0ae24: Waiting
02013e989733: Waiting
c50686409e37: Waiting
be363baee980: Download complete
7b0c919d07c0: Verifying Checksum
7b0c919d07c0: Download complete
4530ca4a6aa0: Download complete
4530ca4a6aa0: Pull complete
7b0c919d07c0: Pull complete
be363baee980: Pull complete
c50686409e37: Verifying Checksum
c50686409e37: Download complete
c50686409e37: Pull complete
b9d9a8a3036e: Verifying Checksum
b9d9a8a3036e: Download complete
b56689c0ae24: Download complete
69c36f50f93d: Verifying Checksum
69c36f50f93d: Download complete
7863da9f2274: Verifying Checksum
466b9fe9cca0: Verifying Checksum
466b9fe9cca0: Download complete
1f4b25b96394: Verifying Checksum
1f4b25b96394: Download complete
e1215d28dca8: Verifying Checksum
e1215d28dca8: Download complete
66fe70d75676: Verifying Checksum
66fe70d75676: Download complete
f4d1a20d6c1a: Verifying Checksum
f4d1a20d6c1a: Download complete
a49e148c97b0: Download complete
d65a4f30a4fa: Verifying Checksum
d65a4f30a4fa: Download complete
b17d3e90cf7f: Verifying Checksum
b17d3e90cf7f: Download complete
02013e989733: Verifying Checksum
02013e989733: Download complete
3ca8e3307e40: Verifying Checksum
3ca8e3307e40: Download complete
90f081dbcb3b: Verifying Checksum
90f081dbcb3b: Download complete
90f081dbcb3b: Pull complete
b9d9a8a3036e: Pull complete
466b9fe9cca0: Pull complete
b56689c0ae24: Pull complete
69c36f50f93d: Pull complete
7863da9f2274: Pull complete
e1215d28dca8: Pull complete
1f4b25b96394: Pull complete
02013e989733: Pull complete
66fe70d75676: Pull complete
f4d1a20d6c1a: Pull complete
a49e148c97b0: Pull complete
d65a4f30a4fa: Pull complete
b17d3e90cf7f: Pull complete
3ca8e3307e40: Pull complete
Digest: sha256:4807cbbdf1e1d3f420bb22060fa379e2fb9c09eb92fe3d17cb479c1573ca4d25
Status: Downloaded newer image for 665840871993.dkr.ecr.us-east-1.amazonaws.com/dev_tools:prod
##> devops/docker_run/entrypoint.sh
UID=1021
GID=1022
# Activate environment
##> devops/docker_run/setenv.sh
# Set PATH
PATH=/app:/app/linters:/app/dev_scripts_devto/notebooks:/app/documentation_devto/scripts:/app/amp/documentation/scripts:/app/amp/dev_scripts/testing:/app/amp/dev_scripts/notebooks:/app/amp/dev_scripts/install:/app/amp/dev_scripts/infra:/app/amp/dev_scripts/git:/app/amp/dev_scripts/aws:/app/amp/dev_scripts:/app/amp:.:/venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
# Set PYTHONPATH
PYTHONPATH=/app:/app/amp:
# Configure env
Testing sudo
/src
[33mWARNING[0m: AWS credential check failed: can't find /home/.aws/credentials file.
[33mWARNING[0m: AWS credential check failed: can't find /home/.aws/config file.
# Check AWS authentication setup
      Name                    Value             Type    Location
      ----                    -----             ----    --------
   profile                       am           manual    --profile

The config profile (am) could not be found
CONTAINER_VERSION=''
BUILD_TAG=''
which python: /venv/bin/python
python -V: Python 3.8.10
/app/devops/docker_run/entrypoint.sh: line 56: docker: command not found
docker -v: 
/app/devops/docker_run/entrypoint.sh: line 57: docker-compose: command not found
docker-compose -v: 
PATH=/app:/app/linters:/app/dev_scripts_devto/notebooks:/app/documentation_devto/scripts:/app/amp/documentation/scripts:/app/amp/dev_scripts/testing:/app/amp/dev_scripts/notebooks:/app/amp/dev_scripts/install:/app/amp/dev_scripts/infra:/app/amp/dev_scripts/git:/app/amp/dev_scripts/aws:/app/amp/dev_scripts:/app/amp:.:/venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PYTHONPATH=/app:/app/amp:
entrypoint.sh: 'pre-commit run -c /app/.pre-commit-config.yaml --files helpers/lib_tasks.py'
Check for merge conflicts................................................[42mPassed[m
Trim Trailing Whitespace.................................................[42mPassed[m
Fix End of Files.........................................................[42mPassed[m
Check for added large files..............................................[42mPassed[m
CRLF end-lines remover...................................................[42mPassed[m
Tabs remover.............................................................[42mPassed[m
autoflake................................................................[42mPassed[m
amp_check_filename.......................................................[42mPassed[m
amp_isort................................................................[42mPassed[m
amp_black................................................................[42mPassed[m
amp_flake8...............................................................[42mPassed[m
amp_doc_formatter........................................................[41mFailed[m
[2m- hook id: amp_doc_formatter[m
[2m- files were modified by this hook[m

[0m[36mINFO[0m: > cmd='/app/linters/amp_doc_formatter.py helpers/lib_tasks.py'

amp_pylint...............................................................[41mFailed[m
[2m- hook id: amp_pylint[m
[2m- exit code: 29[m

[0m[36mINFO[0m: > cmd='/app/linters/amp_pylint.py helpers/lib_tasks.py'
************* Module helpers.lib_tasks
helpers/lib_tasks.py:330: [C0301(line-too-long), ] Line too long (106/100)
helpers/lib_tasks.py:1160: [C0301(line-too-long), ] Line too long (119/100)
helpers/lib_tasks.py:1632: [C0301(line-too-long), ] Line too long (184/100)
helpers/lib_tasks.py:3186: [C0301(line-too-long), ] Line too long (102/100)
helpers/lib_tasks.py:3921: [C0301(line-too-long), ] Line too long (107/100)
helpers/lib_tasks.py:3954: [C0301(line-too-long), ] Line too long (107/100)
helpers/lib_tasks.py:3956: [C0301(line-too-long), ] Line too long (107/100)
helpers/lib_tasks.py:455: [W0621(redefined-outer-name), git_clean] Redefining name 'fix_perms' from outer scope (line 5416)
helpers/lib_tasks.py:2758: [W1203(logging-fstring-interpolation), docker_build_prod_image] Use lazy % formatting in logging functions
helpers/lib_tasks.py:3172: [R1704(redefined-argument-from-local), _find_func_class_uses] Redefining argument with the local name 'regex'
helpers/lib_tasks.py:3197: [W0612(unused-variable), _process_find_results] Unused variable 'file'
helpers/lib_tasks.py:3197: [W0612(unused-variable), _process_find_results] Unused variable 'line_num'
helpers/lib_tasks.py:3197: [W0612(unused-variable), _process_find_results] Unused variable 'line'
helpers/lib_tasks.py:3246: [R1705(no-else-return), find] Unnecessary "elif" after "return"
helpers/lib_tasks.py:3247: [R1704(redefined-argument-from-local), find] Redefining argument with the local name 'mode'
helpers/lib_tasks.py:3324: [E1121(too-many-function-args), find_test_decorator] Too many positional arguments for function call
helpers/lib_tasks.py:3585: [W0621(redefined-outer-name), _run_tests] Redefining name 'git_clean' from outer scope (line 455)
helpers/lib_tasks.py:3630: [W0621(redefined-outer-name), run_fast_tests] Redefining name 'git_clean' from outer scope (line 455)
helpers/lib_tasks.py:3673: [W0621(redefined-outer-name), run_slow_tests] Redefining name 'git_clean' from outer scope (line 455)
helpers/lib_tasks.py:3709: [W0621(redefined-outer-name), run_superslow_tests] Redefining name 'git_clean' from outer scope (line 455)
helpers/lib_tasks.py:3745: [W0621(redefined-outer-name), run_fast_slow_tests] Redefining name 'git_clean' from outer scope (line 455)
helpers/lib_tasks.py:3799: [W0621(redefined-outer-name), run_fast_slow_superslow_tests] Redefining name 'git_clean' from outer scope (line 455)
helpers/lib_tasks.py:4217: [W0621(redefined-outer-name), pytest_repro] Redefining name 'traceback' from outer scope (line 3997)
helpers/lib_tasks.py:4206: [C0207(use-maxsplit-arg), pytest_repro] Use txt.rsplit(start_block, maxsplit=1)[-1] instead
helpers/lib_tasks.py:4201: [W0612(unused-variable), pytest_repro] Unused variable 'i'
helpers/lib_tasks.py:4777: [W1201(logging-not-lazy), gh_login] Use lazy % formatting in logging functions
helpers/lib_tasks.py:4787: [W1201(logging-not-lazy), gh_login] Use lazy % formatting in logging functions
helpers/lib_tasks.py:4897: [R1723(no-else-break), gh_workflow_list] Unnecessary "elif" after "break"

amp_mypy.................................................................[41mFailed[m
[2m- hook id: amp_mypy[m
[2m- exit code: 26[m

[0m[36mINFO[0m: > cmd='/app/linters/amp_mypy.py helpers/lib_tasks.py'
helpers/lib_tasks.py:209: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]
helpers/lib_tasks.py:223: error: Incompatible return value type (got "Optional[Any]", expected "int")  [return-value]
helpers/lib_tasks.py:960: error: Incompatible types in assignment (expression has type "List[str]", variable has type "str")  [assignment]
helpers/lib_tasks.py:1401: error: Returning Any from function declared to return "List[str]"  [no-any-return]
helpers/lib_tasks.py:1448: error: Incompatible types in assignment (expression has type "None", variable has type "bool")  [assignment]
helpers/lib_tasks.py:2139: error: Function is missing a return type annotation  [no-untyped-def]
helpers/lib_tasks.py:3118: error: The return type of a generator function should be "Generator" or one of its supertypes  [misc]
helpers/lib_tasks.py:3172: error: Incompatible types in assignment (expression has type "Pattern[str]", variable has type "str")  [assignment]
helpers/lib_tasks.py:3173: error: "str" has no attribute "search"  [attr-defined]
helpers/lib_tasks.py:3202: error: Argument 1 to "sorted" has incompatible type "List[Tuple[str, int, str, str, str]]"; expected "Iterable[Tuple[str, str]]"  [arg-type]
helpers/lib_tasks.py:3205: error: Incompatible return value type (got "List[Tuple[str, str]]", expected "List[Tuple[Any, ...]]")  [return-value]
helpers/lib_tasks.py:3205: note: "List" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance
helpers/lib_tasks.py:3205: note: Consider using "Sequence" instead, which is covariant
helpers/lib_tasks.py:3205: note: Perhaps you need a type annotation for "filtered_results"? Suggestion: "List[Tuple[Any, ...]]"
helpers/lib_tasks.py:3251: error: Argument 1 to "_find_func_class_uses" has incompatible type "Tuple[str, int, str]"; expected "List[Any]"  [arg-type]
helpers/lib_tasks.py:3261: error: Argument 1 to "_find_short_import" has incompatible type "Tuple[str, int, str]"; expected "List[Any]"  [arg-type]
helpers/lib_tasks.py:3263: error: Argument 1 to "_find_short_import" has incompatible type "Tuple[str, int, str]"; expected "List[Any]"  [arg-type]
helpers/lib_tasks.py:3310: error: Function is missing a type annotation  [no-untyped-def]
helpers/lib_tasks.py:3312: error: unused 'type: ignore' comment
helpers/lib_tasks.py:3324: error: Too many arguments for "_find_test_decorator"  [call-arg]
helpers/lib_tasks.py:4758: error: Function is missing a type annotation  [no-untyped-def]
helpers/lib_tasks.py:4761: error: unused 'type: ignore' comment
helpers/lib_tasks.py:4802: error: Returning Any from function declared to return "Optional[str]"  [no-any-return]
helpers/lib_tasks.py:4843: error: Function is missing a type annotation  [no-untyped-def]
helpers/lib_tasks.py:4848: error: unused 'type: ignore' comment
helpers/lib_tasks.py:5053: error: Returning Any from function declared to return "bool"  [no-any-return]

amp_lint_md..........................................(no files to check)[46;30mSkipped[m
amp_class_method_order...................................................[42mPassed[m
amp_normalize_import.....................................................[41mFailed[m
[2m- hook id: amp_normalize_import[m
[2m- files were modified by this hook[m

[0m[36mINFO[0m: > cmd='/app/linters/amp_normalize_import.py helpers/lib_tasks.py'
17:03:40 - [33mWARN [0m amp_normalize_import.py _remove_import_from_docstring_text:846 The docstring must end with the import line: all comments after the import line will be removed.

amp_format_separating_line...............................................[42mPassed[m
amp_warn_incorrectly_formatted_todo......................................[42mPassed[m
amp_processjupytext......................................................[42mPassed[m
amp_remove_eof_newlines..............................(no files to check)[46;30mSkipped[m
